frontend/packages/core/src/Assets/global.ts

```diff
@@ -32,3 +32,5 @@ export const StyledSVG = styled.svg<{ size?: IconSizeVariant }>(props => ({
 export interface SVGProps extends React.SVGProps<SVGSVGElement> {
   size?: IconSizeVariant;
 }
+
+export const SEVERITIES = ["error", "info", "success", "warning"];
```

Comment:
Lets keep this in the alert component file instead of the global assets. We can also do:

```
export const SEVERITIES = Object.keys(iconMappings);
```